### PR TITLE
[Vitamins Included] Added found design flaws in rev 2.0

### PR DESCRIPTION
**Pull request by Duckle29**
Two design flaws have been found and I detailed them in the documentation for it.

### CODE REVIEW

**Comment by Duckle29** [comment]
I also added some updated 3D models. Printed and confirmed fitting.

**Comment by drashna** [comment]
Merge conflict :(

**Comment by Duckle29** [comment]
well that didn't work... wat

**Comment by Duckle29** [comment]
The merge conflict was due to the _pages being moved. it has been solved :)

**Review by drashna — APPROVED** [review]

File: _i18n/en/keyboards/vitamins_included.md [hunk 1]
```diff
@@ -46,6 +49,7 @@ The PCB files are available [on github](https://github.com/Duckle29/Vitamins-inc
 ![Bottom render](images/bot.png)
 
 ### Top side
+## NOTE: Rev 2.0 has an error in the labeling of the LED_O jumpers on the right half. They are swapped. Just the lables are swapped. Functionally they're identical.
```
Comment (fauxpark) [inline _i18n/en/keyboards/vitamins_included.md]:
```suggestion
#### NOTE: Rev 2.0 has an error in the labeling of the `LED_O` jumpers on the right half. The labels are swapped, but are functionally identical to the previous revision.
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**